script/run-metoffice-daily.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+java -jar kixi.hecuba.weather-0.2.0-SNAPSHOT-standalone.jar -u$khwuser -p$khwpass -c$khwdevicesfile 
```

Comment:
it's useful to do 

`${KHWSER?NOT DEFINED}`

which will error out if the env var isn't defined.